tasks/internet-monitoring.yml

```diff
@@ -13,7 +13,7 @@
 
 - name: Synchronize internet-monitoring directory.
   ansible.posix.synchronize:
-    src: internet-monitoring
+    src: internet-monitoring/
```

Comment:
I believe I fixed this in https://github.com/geerlingguy/internet-pi/commit/4545820436b6978146bd0a422df3896697fcfc85#diff-53441a2c1b0592a9525b6afd8b4b0decf80f2cc52266a79541cd08ab4e0e88f5R17